x-pack/plugins/observability/public/components/custom_threshold/types.ts

```diff
@@ -74,8 +74,6 @@ export interface InfraClientStartDeps {
   uiActions: UiActionsStart;
   unifiedSearch: UnifiedSearchPublicPluginStart;
   usageCollection: UsageCollectionStart;
-  // TODO:: check if needed => https://github.com/elastic/kibana/issues/159340
```

Comment:
I checked SLO burn rate, and we don't use specific telemetry in our rules, so it should be safe to not copy this here.